docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -6337,6 +6337,300 @@ UPDATE persnl.job
 SET jobdesc = RIGHT (jobdesc, 12);
 ```
 
+<<<
+[[rollup_function]]
+== ROLLUP Function
+
+The ROLLUP function calculates multiple levels of subtotals aggregating from right to left through the comma-separated list of columns, and provides a grand total. It is a an extension to the `GROUP BY` clause and can be used with `ORDER BY` to sort the results.
+
+```
+SELECT…GROUP BY ROLLUP (column 1, [column 2,]…[column n])
+```
+
+ROLLUP generates n+1 levels of subtotals and grand total, where n is the number of the selected column(s).
```

Comment:
Suggested wordsmith "ROLLUP generates n+1 levels of subtotals, including a grand total, ..."

Reply:
Thanks Dave, your comment has been incorporated :)